src/AudioSystem.h

```diff
@@ -6,20 +6,24 @@
 #include "EventSystem.h"
 #include "SDL_mixer.h"
 
-class AudioSystem : public EventSystem<AudioSystem> {
-
-    bool init_successful_ = false;
-    std::unordered_map<std::string, Mix_Chunk*> sound_assets_;
-    std::unordered_map<std::string, Mix_Music*> music_assets_;
+enum class SoundType {BEAT};
+enum class MusicType {BEAT};
```

Comment:
Can this be a separate file? Also sorry I changed my mind, I think MusicAsset, and SoundAsset is better. They're not really "types"